lib/ability.rb

```diff
@@ -86,12 +113,13 @@ def initialize(user, resource, controller)
 
       if user.facility_director_of?(resource)
         can [ :activate, :deactivate ], ExternalService
+        can :disputed, [Order, Reservation]
```

Comment:
Yes. I had gone with `:manage` instead of `:disputed` when it looked like it was redundant in 6ec2da2, but I just changed it to check `:disputed` (including in the drop-down, though only against `Order`).